src/components/experiment-tracking/run-metadata/run-metadata.js

```diff
@@ -102,6 +103,27 @@ const RunMetadata = ({
     setShowRunDetailsModal(true);
   };
 
+  const getNotesByRunId = (runId) => {
+    if (runsMetaData[runId]) {
+      return runsMetaData[runId][RUN_NOTES] || '';
+    }
+    return '';
```

Comment:
Empty line before `return` statements.
```suggestion
    if (runsMetaData[runId]) {
      return runsMetaData[runId][RUN_NOTES] || '';
    }
    
    return '';
```

Reply:
done.